src/vs/platform/terminal/common/terminalProfiles.ts

```diff
@@ -93,8 +93,8 @@ export function terminalIconsEqual(iconOne?: TerminalIcon, iconTwo?: TerminalIco
 	if (ThemeIcon.isThemeIcon(iconOne) && ThemeIcon.isThemeIcon(iconTwo)) {
 		return iconOne.id === iconTwo.id && iconOne.color === iconTwo.color;
 	}
```

Comment:
I think this monstrosity was my doing, but we could go with `a` and `b` instead of `iconOne` and `iconTwo` all over the place since the context IE `terminalIconsEqual` makes it clear what we're comparing already.

Reply:
https://github.com/microsoft/vscode/pull/159667